data/sync_covid_testing.py

```diff
@@ -0,0 +1,95 @@
+"""
+Pull data from MOPS COVID Dashboard and upload to S3
```

Comment:
Is the mayor's office still updating this?

Reply:
they are....probably that same spreadsheet is feeding into the tableau dashboard